test/fixtures/render/content-type.apib

```diff
@@ -0,0 +1,14 @@
+# GET /
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + user: pksunkara
+
+# GET /hal
++ Response 200 (application/hal+json)
+    + Attributes
+        + user: pksunkara
+
+# GET /typo
++ Response 200 (application/haljson)
```

Comment:
As you see the regex, I am only checking for the aforementioned patterns. This weird example is there to show that this content type will not generate a json render. Please check the output below